Floofbot/Modules/UserAssignableRoleCommands.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using Discord;
+using System.Threading.Tasks;
+using Discord.Commands;
+using System.Text.RegularExpressions;
+using Floofbot.Services.Repository;
+using Floofbot.Services.Repository.Models;
+using System.Linq;
+using Discord.Addons.Interactive;
+using Discord.WebSocket;
+using Serilog;
+
+namespace Floofbot.Modules
+{
+    public class UserAssignableRoleCommands : InteractiveBase
```

Comment:
What's the module that shows up when doing .help?
I wonder if we need a summary for the overall module